addons/hw_escpos/escpos/printer.py

```diff
@@ -83,7 +83,7 @@ def close(self):
 
     def _raw(self, msg):
         """ Print any command sent in raw format """
-        if len(msg) != self.device.write(self.out_ep, msg, self.interface, timeout=5000):
+        if len(msg) != self.device.write(self.out_ep, msg, self.interface): #, timeout=5000
```

Comment:
why?

Reply:
Indeed, I need to check.  It gave errors, but maybe for another reason than the api would have changed between python2/3 (could not find this back easily)

Reply:
putting timeout back, it works, so I will put it back.